### PR TITLE
Fix build on Android Studio 3.6

### PR DESCRIPTION
**Pull request by robinlinden**
This required updating a bunch of things that were no longer supported
and removing the minVersion from the android manifest.

Probably not only a problem with Android Studio, but that's what I was trying to build toktok-android with. Running gradle via the gradlew wrapper also seemed to fail.

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/toktok-android/53)

### CODE REVIEW

**Review by iphydf — APPROVED** [review]

**Review by iphydf — CHANGES_REQUESTED** [review]
Un-LGTM.

File: src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="im.tox.toktok">
-    <uses-sdk android:minSdkVersion="22"
```
Comment (iphydf) [inline src/main/AndroidManifest.xml:4]: Why did you remove this? I'm getting warnings without it.

**Review by iphydf — CHANGES_REQUESTED** [review]

File: src/main/AndroidManifest.xml (hunk 1 above)
Comment (robinlinden) [reply src/main/AndroidManifest.xml:4]: The Gradle sync thing fails with it there:
```
ERROR: The minSdk version should not be declared in the android manifest file. You can move the version from the manifest to the defaultConfig in the build.gradle file.
Remove minSdkVersion and sync project
Affected Modules: toktok-android
```

Comment (robinlinden) [reply src/main/AndroidManifest.xml:4]: I added them to the BUILD.bazel, so now that should work again.

**Review by iphydf — APPROVED** [review]
